main.go

```diff
@@ -98,18 +98,21 @@ func main() {
 	// define environment
 	var env gateways.Environment
 	var confFileName string
+	var configURL string
 	if ENV == "prod" {
 		env = gateways.Production
 		zerolog.SetGlobalLevel(zerolog.InfoLevel)
 		confFileName = "config.yaml"
+		configURL = "https://device-config-prod.dimo-builder.workers.dev/"
 	} else {
 		env = gateways.Development
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
 		confFileName = "config-dev.yaml"
+		configURL = "https://device-config.dimo-builder.workers.dev/"
```

Comment:
are these urls final?
I would be more explicit on naming eg. `device-config-dev` vs `device-config-prod`, normally otherwise the prod environment has no `-prod` and only the dev environment is appended a `-dev`

Reply:
No, these urls are not final, it is wip. I need help on cloudflare side to make better urls

Reply:
The new URLs for retrieving configuration are:
1) https://device-config-dev.dimo.xyz - for dev
2) https://device-config.dimo.xyz - for prod